scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,111 @@
+      --set scanner.resources.limits.memory=2500Mi \
+      --set scanner.resources.limits.cpu=2000m
+fi
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Forwarding ports to central"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with shared secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+installflags=("-f stackrox-init-bundle.yaml" "--set clusterName=\"my-secured-cluster\"")
+if $SMALL_INSTALL; then
+    installflags+=("--set sensor.resources.requests.memory=500Mi")
+    installflags+=("--set sensor.resources.requests.cpu=500m")
+    installflags+=("--set sensor.resources.limits.memory=500Mi")
+    installflags+=("--set sensor.resources.limits.cpu=500m")
+    echo "Installing the first small secured cluster"
+else
+    echo "Installing the first secured cluster"
+fi
+
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services ${installflags[*]}
+
+echo "Logging into StackRox in the browser"
+
+curl_status=0
+target_url="$(curl -sSkf -u "admin:${STACKROX_ADMIN_PASSWORD}" -o /dev/null -w '%{redirect_url}' "https://localhost:8000/sso/providers/basic/4df1b98c-24ed-4073-a9ad-356aec6bb62d/challenge?micro_ts=0")" || curl_status=$?
+if [[ "$curl_status" -eq 0 && -n "$target_url" ]]; then
+    if [[ -n "$target_url" ]]; then
+        auth_error="$(sed -E '/^.*#error=([^&]+)(&.*)?$/!d;s//\1/' <<<"$target_url" | tr '+' ' ')"
+        if [[ -z "$auth_error" ]]; then
+            echo "Logging you in via ${target_url} ..."
+            platform="$(uname)"
+            if [[ "$platform" == "Linux" ]]; then
+                xdg-open "${target_url}" >/dev/null &
+                echo ""
+                echo "You may access the dashboard via https://localhost:8000/main/dashboard now."
+            elif [[ "$platform" == "Darwin" ]]; then
+                open "${target_url}" &
+                echo ""
+                echo "You may access the dashboard via https://localhost:8000/main/dashboard now."
+            else
+                echo "Unsupported platform '$platform', please open $@ in a browser" 1>&2
+            fi
+        else
+            echo "Authentication error: ${auth_error}" 1>&2
+        fi
+    else
+        echo "UNEXPECTED: Could not determine target URL" 1>&2
+    fi
+else
+    echo "Could not determine login URL for basic auth user" 1>&2
+fi
+
+echo "Consult these documents for additional information on configuring your Helm installation:
+https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html
+https://docs.openshift.com/acs/installing/installing_helm/install-helm-customization.html
+
+\$STACKROX_ADMIN_PASSWORD = $STACKROX_ADMIN_PASSWORD
```

Comment:
We want to display `STACKROX_ADMIN_PASSWORD=$STACKROX_ADMIN_PASSWORD`, so to make it copy-pastable, I suggest to:
- Remove `\$`
- Remove spaces around `=`

Reply:
Fair